src/graphnet/data/dataset.py

```diff
@@ -598,36 +598,51 @@ def _create_graph(
 
         # Additionally add original features as (static) attributes
         for index, feature in enumerate(graph.features):
-            graph[feature] = graph.x[:, index].detach()
```

Comment:
Ahh, this is because the Kaggle-dataset feature `x` conflicts with the torch-geometric built-in feature `x` referring to the features tensor?

Reply:
Yes.